lmdk/cmake/ldscripts/bss_linker_script.txt

```diff
@@ -16,5 +16,18 @@ SECTIONS {
         _next_end = ABSOLUTE(.);
         *(.bss)
         *(.bss.*)
+
+        /* Heap memory */
+        . = ALIGN(4);
+        __stack = .;
+        _end = .;
```

Comment:
are these symbols used?

Reply:
Start of heap memory is marked by the `_end` symbol.